static/app/components/menuListItem.tsx

```diff
@@ -259,11 +259,13 @@ export const InnerWrap = styled('div', {
   box-sizing: border-box;
 
   font-size: ${p => p.theme.form[p.size ?? 'md'].fontSize};
+  color: ${getTextColor};
 
-  &,
   &:hover {
     color: ${getTextColor};
+    background: ${p => p.theme.backgroundSecondary};
   }
+
```

Comment:
Adding this leads to confusing behaviors like here:
<img width="717" alt="image" src="https://github.com/getsentry/sentry/assets/44172267/ff51ffc5-38ff-4436-bc4b-5ee98c76a494">
…where the second item was focused on via the keyboard, and the fourth item was focused on by hovering over it.

Reply:
In the future we can better address this by implementing a [combobox](https://codesandbox.io/p/sandbox/hardcore-moon-xzc4r?file=%2Fsrc%2FSearchAutocomplete.tsx) pattern, but that's way more complicated and should be left for later.